argocd/argoapp-smallstep.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: "smallstep"
+  namespace: "argocd"
+spec:
+  destination:
+    namespace: smallstep
+    server: https://kubernetes.default.svc
+  project: default
+  syncPolicy:
+    automated:
+      prune: true
+    syncOptions:
+      - CreateNamespace=true
+  source:
+    repoURL: https://github.com/advatys/smallstep-helm-charts.git
+    targetRevision: openshift
+    path: step-certificates
+    helm:
+      values: |
+        ca:
+          bootstrap:
+            # Add script snippets here to be executed after the step ca init has been run
+            postInitHook: |-
+              step ca provisioner add Google --type oidc --ca-config $(step path)/config/ca.json \
+              --client-id 650445034027-jsjdrkiskeq9ke99ud2rqkst82ft8uch.apps.googleusercontent.com \
```

Comment:
this is secret information and should not be here

Reply:
it's example from https://smallstep.com/docs/step-ca/configuration#provisioners

Reply:
removed